modular_citadel/code/modules/client/loadout/__donator.dm

```diff
@@ -531,10 +531,14 @@
 	path = /obj/item/clothing/under/custom/vest
 	ckeywhitelist = list("maylowfox")
 
-/datum/gear/donator/exo
-	name = "exo frame"
-	slot = SLOT_WEAR_SUIT
-	path = /obj/item/clothing/suit/custom/exo
-	ckeywhitelist = list("jesterz7")
-
+/datum/gear/donator/smask
+	name = "Strange Metal Mask"
+	slot = SLOT_WEAR_MASK
+	path = /obj/item/clothing/mask/smask
+	ckeywhitelist = list("sneka")
 
+/datum/gear/donator/smask
+	name = "Strange Metal Mask"
+	slot = SLOT_WEAR_MASK
+	path = /obj/item/clothing/mask/smask
+	ckeywhitelist = list("sneka")
```

Comment:
you've added the datum/gear/donator/smask twice and removed /datum/gear/donator/exo